CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 1.6.5
```

Comment:
Since this is a new API surface are it should be a feature bump

```suggestion
## 1.7.0
```

Reply:
Updated to `1.7.0`.